### PR TITLE
Fix problem with expand option

### PR DESCRIPTION
**Pull request by seizadi**
The expand option was broken, it did not use go-bindata package and also the paths were broken when you used go-bindata. I think it was tested in a development environment where all template files were available and when run in a standalone binary would break.

In the process of fixing and building a new binary found that there was regression and
Resolving imports... stage broke with EOF exception in the templates.Process()
>> 		withImports, err := imports.Process(path, nil, nil)

I fixed this by upgrading from 2018 era packages to current so the tools package is now
	golang.org/x/tools v0.0.0-20200528185414-6be401e3f76e

I switched the project to use Go Module.

### CODE REVIEW

**Comment by drewwells** [comment]
Is this code being built so tests next time?

File: .travis.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@ services:
   - docker
 
 go:
-  - "1.13.x"
+  - "1.14.x"
 
 env:
   global:
-    - HELM_VERSION="v2.14.3"
+    - HELM_VERSION="v2.16.7"
```
Comment (drewwells) [inline .travis.yml:11]: We currently test/package everything with 2.14.3. Is this change required? Should it be rolled out everywhere?

Comment (seizadi) [reply .travis.yml:11]: Not required, I was updating all the package and updated that one as well. At this point we should be migrating everything to 3.x, but if 2.14 is the standard, I can set it back.

Comment (seizadi) [reply .travis.yml]: Set this to v2.14.3, talked to MikeL about this issue. He will coordinate moving the Application Teams from 2.14 to 2.16 and long term to Helm 3.
